options/locale/locale_en-US.ini

```
@@ -1398,6 +1398,13 @@ issues.content_history.created = created
 issues.content_history.delete_from_history = Delete from history
 issues.content_history.delete_from_history_confirm = Delete from history?
 issues.content_history.options = Options
+issues.confidential = Confidential
+issues.confidential.off = Make issue public
+issues.confidential.off_popup = By clicking this button, you will make this issue and comments public to everyone who can read this repo.
```

```suggestion
issues.confidential.off_popup = By clicking this button, this issue and comments will be visible to everyone with read access to this repositry.
```

I don't really like this "By clicking this button" because it refers to a specifc UI action. Maybe word it more generically like "Please confirm...".

---

> Maybe word it more generically like "Please confirm...".

I couldn't come with a better idea so left it as-is, for anyone to improve it.